src/components/ProfileVisualization.js

```diff
@@ -98,15 +106,17 @@ const ProfileVisualization = (props) => {
   }
 
   const consolidateData = () => {
-    const cleanedUserData = filteredRepos.map((repo, index) => {
-      return {
-        name: repo.name,
-        branches: branchNames[index],
-        lifespan: lifespans[index],
-        languages: repoLangs[index]
-      }
-    })
-    setCleanUserData(cleanedUserData);
+    if (repoLangs.length) {
```

Comment:
good temporary solution, though I would love to figure out why our async `.map`s won't behave!